clients/venice-push-job/src/main/java/com/linkedin/venice/hadoop/VenicePushJob.java

```diff
@@ -2307,12 +2311,26 @@ void validateValueSchema(
         .info("Validating value schema: {} for store: {}", pushJobSchemaInfo.getValueSchemaString(), setting.storeName);
 
     SchemaResponse getValueSchemaIdResponse;
-
     if (setting.enableWriteCompute) {
```

Comment:
So the user has to specify `venice.write.compute.enable` to enable write compute in inc push job, it is fine for now, and we need to let the new users be aware of this config.

Reply:
Yeah, it is the current behavior. I don't plan to change it inside this PR as it could be very big change but we could migrate this out in a separate PR.